plugins/telemetry-plugin/webpack.configold..js

```diff
@@ -0,0 +1,51 @@
+/*********************************************************************
```

Comment:
I would say this file is not used ? (it contains old in its name)

Reply:
@davidfestal does this file still has some value or should we simply drop it?

Reply:
I didn't remove it because I was more or less expecting it to become a front-end plugin again when front-end plugins are supported again.
But in the meantime I'm OK to remove this file.

Reply:
removed